samples/net/openthread/ot_cli/prj.conf

```diff
@@ -0,0 +1,59 @@
+# Generic networking options
+CONFIG_NETWORKING=y
+CONFIG_NET_UDP=y
+CONFIG_NET_IPV6=y
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_POSIX_NAMES=y
+CONFIG_NET_SOCKETS_POLL_MAX=4
+CONFIG_NET_CONNECTION_MANAGER=y
+
+# Kernel options
+CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_ENTROPY_GENERATOR=y
+CONFIG_INIT_STACKS=y
+
+# Logging
+CONFIG_NET_LOG=y
+CONFIG_LOG=y
+CONFIG_NET_STATISTICS=y
+CONFIG_PRINTK=y
+CONFIG_OT_COMMAND_LINE_INTERFACE_LOG_LEVEL_DBG=y
+
+# Network shell
+CONFIG_NET_SHELL=y
+
+CONFIG_CPLUSPLUS=y
+
+CONFIG_NET_CONFIG_SETTINGS=y
+CONFIG_NET_IPV6_NBR_CACHE=n
+CONFIG_NET_IPV6_MLD=n
+CONFIG_NET_CONFIG_NEED_IPV6=y
+CONFIG_NET_CONFIG_INIT_TIMEOUT=3
```

Comment:
Couldn't this be 0 for the CLI app? It's not taking any action on its own after startup.